### ISSUE TITLE
BadDeserialize error

### ISSUE THREAD

**Issue by skoky** [rektdeckard/hues#2]
I am running this example code with version `0.1.0`:

```
use std::fs;
use hues::prelude::*;

#[tokio::main]
async fn main() -> Result<(), HueAPIError> {
    // Construct a Bridge when IP and App Key are known
    let key = fs::read_to_string("hue_key").expect("Missing key file");
    let bridge = Bridge::new([10, 0, 10, 3], key);
    // Refresh it to fetch the current state of all resources
    bridge.refresh().await?;

    // Toggle the power state of a Room named "office", if it exists
    if let Some(office) = bridge.rooms().iter().find(|r| r.name() == "office") {
        office.toggle().await?;
    }
    Ok(())
}
```

and I am getting this error:

```
Err E reqwest::Error { kind: Decode, source: Error("unknown variant `living_room`, expected one of `attic`, `balcony`, `barbecue`, `bathroom`, `bedroom`, `carport`, `closet`, `computer`, `dining`, `downstairs`, `driveway`, `front_door`, `garage`, `garden`, `guest_zone`, `gym`, `hallway`, `home`, `kids_bedroom`, `kitchen`, `laundry_zone`, `living_zone`, `lounge`, `man_cave`, `music`, `nursery`, `office`, `other`, `pool`, `porch`, `reading`, `recreation`, `staircase`, `storage`, `studio`, `terrace`, `toilet`, `top_floor`, `tv`, `upstairs`", line: 1, column: 11345) }
error decoding response body: unknown variant `living_room`, expected one of `attic`, `balcony`, `barbecue`, `bathroom`, `bedroom`, `carport`, `closet`, `computer`, `dining`, `downstairs`, `driveway`, `front_door`, `garage`, `garden`, `guest_zone`, `gym`, `hallway`, `home`, `kids_bedroom`, `kitchen`, `laundry_zone`, `living_zone`, `lounge`, `man_cave`, `music`, `nursery`, `office`, `other`, `pool`, `porch`, `reading`, `recreation`, `staircase`, `storage`, `studio`, `terrace`, `toilet`, `top_floor`, `tv`, `upstairs` at line 1 column 11345
Error: BadDeserialize
```

can you advice if my setup or version of Hue bridge is incompatible or how to fix? Thanks

**Comment by rektdeckard** [comment]
Hi, thanks for the report! This lib is very much alpha, and the official documentation is not always accurate. I'll add `living_room` as a possible room variant, and also improve serialization of unknown values here.

For my reference, can you let me know your bridge version, and the version of the firmware it is running?

**Comment by rektdeckard** [comment]
Should be fixed in `0.1.2`. LMK if it works for you.

**Comment by skoky** [comment]
yes, its fixed and works Thanks
